build.sbt

```diff
@@ -3,7 +3,7 @@ import scala.collection.mutable
 def scala212 = "2.12.12"
 def scala211 = "2.11.12"
 def scala213 = "2.13.3"
-def scala3 = List("3.0.0-M1")
+def scala3 = List("3.0.0-M2")
```

Comment:
It seems much more problematic to cross build and we can easily just use another version in Metals. Especially with some much changed. It will be much easier after I finish https://github.com/lampepfl/dotty/pull/10172